ttl/ttlworker/session.go

```diff
@@ -152,7 +152,7 @@ type ttlTableSession struct {
 	expire time.Time
 }
 
-func (s *ttlTableSession) ExecuteSQLWithCheck(ctx context.Context, sql string) (rows []chunk.Row, shouldRetry bool, err error) {
+func (s *ttlTableSession) ExecuteSQLWithCheck(ctx context.Context, sql string) (rows []chunk.Row, _ bool, err error) {
```

Comment:
Please keep the output variable or remove all output variables and declare `rows` before `s.RunInTxn`

Reply:
done